sorc/ufs_noaacloud.intel.diff

```diff
@@ -0,0 +1,10 @@
+--- ufs_noaacloud.intel.lua	2024-10-03 15:54:33.334583588 +0000
```

Comment:
Will the need for this be resolved before this gets merged?

Reply:
This diff should not be committed.  The ufs-weather-model needs to be able to be built on AWS without the need for a diff in the global-workflow